Apps/AdminWebClient/src/appsettings.hgpoc.json

```diff
@@ -0,0 +1,13 @@
+{
+    "Logging": {
+        "LogLevel": {
+            "HealthGateway": "Trace"
+        }
+    },
+    "OpenIdConnect": {
+        "Authority": "https://dev.oidc.gov.bc.ca/auth/realms/ff09qn3f"
```

Comment:
I assume that our dev and poc HGW environments are using the same dev.oidc right?

Reply:
They are indeed!